packages/paste-website/src/pages/components/form/index.mdx

```diff
@@ -0,0 +1,690 @@
+  noInline
+>
+  {singleColumnForm}
+</LivePreview>
+
+## Guidelines
+
+### About Form
+
+Use the Form component to arrange a layout of form elements with preset spacing.
+The Form component renders an HTML form element and comes with default behavior from the browser ([learn more about HTML form elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form)). Form elements are displayed vertically by default, but also support horizontal and 2-column layouts.
+
+Common form components include [Buttons](/components/button), [Comboboxes](/components/combobox), [Checkboxes](/components/checkbox), [Date Picker](/components/date-picker), [File Picker](/components/file-picker), [Form Pill Groups](/components/form-pill-group), [Help Text](/components/help-text), [Inputs](/components/input), [Label](/components/label), [Radio Group](/components/radio-group), [Selects](/components/select), [Switch](/components/switch), [Textarea](/components/textarea), and [Time Picker](/components/time-picker).
+
+Designing a good form requires making decisions about composition, sequence, form elements, copy, and feedback.
+
+- Forms often have a header, with a Heading and Paragraph that explains the purpose of the form or form section. The header can also include additional contextual components like Alerts or Callouts for expressing errors or other important information.
+- Forms can include [Form Actions](/components/form#form-actions), often with a primary Button ordered first, then a secondary Button second.
+- More complex forms should be split into sections, known as fieldsets. Form sections are separated with a Separator. Sections often have their own header, with a Heading, known as a legend, and can also include a Paragraph to explain the section.
+
+### Accessibility
+
+#### Order
+
+- Order forms logically, in sequential order, following the natural flow of the user's language preference. Typically top to bottom, and left to right for Latin-based languages.
+- Only use a single Form per page.
+- Don’t use CSS to control element order. Form elements should be displayed in the same order in the HTML as they appear on screen, as screen readers announce form elements in the order they appear in the HTML.
+
+#### Content
+
+- Provide accessible and clear labels, placeholder text, and help text.
+  - Labels should clearly describe the value being requested.
+  - Don't use placeholder text as a replacement for labels. It can be used to provide an example to users, but will disappear from the field when a user enters text. It's also not broadly supported by assistive technologies and won't display in older browsers.
+  - Labels should not run onto two lines.
+- Use fieldsets and legends.
+  - When you need to group related form elements, use a Form Section, which renders a HTML fieldset. To add clarity to the section, use a Form Section Heading, which renders a HTML legend, to clearly describe the group. It should be concise and descriptive.
```

Comment:
@shleewhite suggestion: could we link "Form Section" to the "Sections" section (lol).